implementations/rust/ockam/ockam_command/tests/bats/unit.bats

```diff
@@ -225,6 +225,12 @@ teardown() {
 
 }
 
+@test "tcp - create a tcp listner " {
+  run "$OCKAM" tcp-listener create 127.0.0.1:55001
+  assert_output --regexp '[{"/dnsaddr/localhost/tcp/[[:digit:]]+/service/*"}]'
+
+}
```

Comment:
The test fails because there is no node. You have to create a node first.
```bash
run "$OCKAM" node create
```

Also, can you fix the trailing space on the test name and the blank line (231)?